entity-types/infra-rabbitmqqueue/definition.yml

```diff
@@ -1,7 +1,51 @@
 domain: INFRA
 type: RABBITMQQUEUE
+
+synthesis:
+  # Apart from this rule RABBITMQQUEUE entities are also created from infra pipeline.
+  rules:
+  - compositeIdentifier:
+      separator: ":"
+      attributes:
+      - rabbitmq.node.name
```

Comment:
I imagine that the queue name could be repeated across instances, what's about the node.name? it is almost unique?